src/lib/y2network/interface_config_builder.rb

```diff
@@ -436,11 +439,25 @@ def find_interface
       yast_config.interfaces.by_name(name)
     end
 
+    # Returns the interfaces collection
+    #
+    # @return [Y2Network::InterfacesCollection]
+    def interfaces
+      yast_config.interfaces
```

Comment:
when you add this method ( good one ), please also try to replace all `yast_config.interfaces` in file.